relay-server/src/metric_stats.rs

```diff
@@ -66,6 +77,35 @@ impl MetricStats {
             .send(MergeBuckets::new(scoping.project_key, vec![volume]));
     }
 
+    /// Tracks the cardinality of a metric.
+    pub fn cardinality(
```

Comment:
Wdyt about `track_cardinality`? I find it more predictable from the API's user pov, even if it's similar to the `track` above.

Reply:
+1, I left a similar comment here: https://github.com/getsentry/relay/pull/3342/files#r1542984095

Reply:
@jjbayer sorry, I remember renaming it not sure what happened to it :(